ci/glci/model.py

```diff
@@ -476,7 +476,8 @@ class OnlineReleaseManifest(ReleaseManifest):
     s3_key: str
     s3_bucket: str
     test_result: typing.Optional[ReleaseTestResult]
-    logs: typing.Optional[str]
+    logs: typing.Optional[typing.Union[S3_ReleaseFile, str]] = None
+            # Note Union can be removed after all old manifests have been removed
```

Comment:
nit: the indentation looks a bit strange to me. rather mv the comment to the same indentation level as the attrs + maybe mv before the attr the comment relates to?

Reply:
Done